stepper-2.0.md

```diff
@@ -181,12 +210,15 @@ Stepper instances that have been created with the stepper configuration command
 2  multi to command                         (0x21)
 3  group number                             (0-127)
 4  member number                            (0-9)
```

Comment:
It seems like member number could be dropped here. MultiStepper To command would expect to receive as many blocks of "num steps" as there are members in the group, and the index of each block implicitly specifies the group member.

As @dtex suggested, in cases where only a subset of the steppers needs to be controlled, a new group can be created with only the subset as members.